content/html/concepts/tags/terms/img/img.md

```diff
@@ -21,12 +21,12 @@ Used to add an image to the document. It can also be used inside an anchor tag t
 
 `<img>` is a tag that has no closing tag, meaning that it cannot have tags inside of it. It uses attributes to set properties, some common attributes are:
 
-- `src`: the URL location of the image
-- `alt`: alternative text used by screen readers and when the image does not load
-- `height`: the height of the image, in pixels
-- `width`: the width of the image, in pixels
+- `src`: the URL or filename of the image that supports a litany of formats (GIF, JPEG, PNG, etc.).`
+- `alt`: alternative text used by screen readers and when the image does not load.
+- `height`: the height of the image, measured in pixels.
+- `width`: the width of the image, measured in pixels.
 
-## Example
+## Example 1
```

Comment:
Are you planning on adding more examples? I'm not opposed to that, but if we're not adding another "Example 2" below, let's change this back to "Example" 👍